### ISSUE TITLE
Work with subfolder installs

### ISSUE THREAD

**Issue by zoerooney** [Automattic/Picard#22]
This ties into your item about working with different permalink structures (listed in the ReadMe) but currently no content displays if WordPress is installed in a subdirectory.

For example, on my local machine I have a bunch of WordPress installations all within a single parent directory accessed something like:

127.0.0.1/core/
127.0.0.1/dev/

Picard didn't work to display post/ page content when installed into one of those, but it does work when installed in the root install at 

127.0.0.1

**Comment by nb** [comment]
Thanks for the report, @zoerooney, it looks we’re using root-based URLs in `components/router/router.jsx` like `/wp-json/…`.

We accept pull requests :-)

**Comment by kwight** [comment]
You're right, I think we need some sort of config file to handle situations like this, maybe similar to https://github.com/kwight/tango/blob/master/js/config.js

**Comment by zoerooney** [comment]
I'm just coming to this for the first time today via @jacklenox's presentation at LoopConf, but I'll do what I can to contribute ongoing :)

**Comment by nb** [comment]
@kwight isn’t there a way to pass the base API URL from the WordPress install and [avoid options](https://wordpress.org/about/philosophy/#decisions)?

**Comment by jacklenox** [comment]
@nb I think this is possible by passing in the permalink settings via `wp_localize_script()` when the JS is first loaded. That's my current plan for trying to deal with this particular issue.

**Comment by artpi** [comment]
Working on it right now. Will submit pull request shortly.
